cli/root.go

```diff
@@ -107,9 +108,32 @@ func Root() *cobra.Command {
 	cmd.PersistentFlags().Bool(varNoOpen, false, "Block automatically opening URLs in the browser.")
 	_ = cmd.PersistentFlags().MarkHidden(varNoOpen)
 
+	// Cobra automatically uses "-v" and "--version" for printing the version on
+	// the root cmd. If we decide to use "-v" as a verbose flag on the root,
+	// then that will be a behavior change. So we should just reserve "-v"
+	// and make users use "coder --version" or "coder version"
+	cmd.Flags().BoolP("placeholder", "v", false, "This flag is a placeholder to make the cobra version flag work appropriately")
+	_ = cmd.Flags().MarkHidden("placeholder")
```

Comment:
The fact you cannot control this in any other way in cobra is quite frustrating.

Reply:
Nice thinking to reserve it! 💪🏻 (And agreed on the frustrating part.)